qutebrowser/config/configtypes.py

```diff
@@ -1223,14 +1224,68 @@ class AcceptCookies(BaseType):
                                ('never', "Don't accept cookies at all."))
 
 
-class ConfirmQuit(BaseType):
+class ConfirmQuit(List):
 
     """Whether to display a confirmation when the window is closed."""
 
+    typestr = 'string-list'
+
     valid_values = ValidValues(('always', "Always show a confirmation."),
                                ('multiple-tabs', "Show a confirmation if "
                                                  "multiple tabs are opened."),
+                               ('downloads', "show a confirmation if downloads"
+                                             "are running"),
                                ('never', "Never show a confirmation."))
+    # Values that can be combined with commas
+    combinable_values = ('multiple-tabs', 'downloads')
+
+    def transform(self, value):
+        # Backward compatible
+        if value == 'never':
+            return value
+        # Split configuration string into list
+        else:
+            return super().transform(value)
+
+    def validate(self, value):
+        values = self.transform(value)
+        # Backward compatibility
+        if values == 'never':
+            return
+        # Never can't be set with other options
+        elif 'never' in values and isinstance(values,
+                                              list) and len(values) > 1:
+            raise configexc.ValidationError(value, "List cannot contain never!")
+        # Always can't be set with other options
+        elif 'always' in values and isinstance(values,
+                                               list) and len(values) > 1:
+            raise configexc.ValidationError(value,
+                                            "List cannot contain always!")
+        # Values have to be valid
+        elif not set(values).issubset(set(self.valid_values.values)):
+            raise configexc.ValidationError(value, "List contains invalid"
+                                                   " values!")
+        # List can't have duplicates
+        elif len(set(values)) != len(values):
+            raise configexc.ValidationError(value, "List contains duplicate"
+                                                   " values!")
+
+    def complete(self):
+        permutations = []
+        # Generate combinations of the options that can be combined
+        for size in range(2, len(self.combinable_values) + 1):
+            permutations = permutations + list(
+                itertools.permutations(self.combinable_values, size))
```

Comment:
I'm not sure if using `permutations` instead of `combinations` here really makes sense. If we add a third and maybe fourth option in the future, the list will be hard to scan and the outcomes will basically be the same. It would make sense from the "type a part and complete" point of view, but not so much from the "select from a list of possibilities" point of view. What do you think?